sdk/eventhub/Azure.Messaging.EventHubs.Processor/tests/BlobsCheckpointStore/BlobsCheckpointStoreLiveTests.cs

```diff
@@ -41,14 +40,14 @@ await using (StorageScope storageScope = await StorageScope.CreateAsync())
                 var storageConnectionString = StorageTestEnvironment.StorageConnectionString;
                 var containerClient = new BlobContainerClient(storageConnectionString, storageScope.ContainerName);
 
-                var partitionManager = new Processor.BlobsCheckpointStore(containerClient);
+                var partitionManager = new BlobsCheckpointStore(containerClient);
```

Comment:
Not a big thing, but maybe we should mass rename `var partitionManger` to `var checkpointStore` in this class?

Reply:
Agreed.

Reply:
Changed it. Will we be calling the type `CheckpointStore` from now on? To me, the name feels like it handles only checkpoints and not ownership.